### PR TITLE
Initial GSoC Ideas page

### PR DESCRIPTION
**Pull request by tracymiranda**

### CODE REVIEW

**Review by markjacksonfishing — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+CDF is planning to participate in GSoC 2020. 
+
+## Org Admins
+
+
+## Project Ideas 
+If you are a project maintainer and consider mentoring during the GSoC 2020 cycle, please, submit your ideas below using the template.
+
+### Jenkins
+Jenkins project ideas
```
Comment (oleg-nenashev) [inline README.md]:
```suggestion
This year the Jenkins project participates in GSoC as a separate GSoC mentoring organization.
You can file the organizations profile and application guidelines [here](https://summerofcode.withgoogle.com/organizations/4945163270488064/).
List of the project ideas is available on the [Jenkins website](https://jenkins.io/projects/gsoc/2020/project-ideas/).
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+### Jenkins X 
+Jenkins X project ideas
```
Comment (oleg-nenashev) [inline README.md]: May change later, good enough for now

```suggestion
This year Jenkins X project participates in GSoC under umbrella of the Jenkins project.
You can file the organizations profile and application guidelines [here](https://summerofcode.withgoogle.com/organizations/4945163270488064/).
List of the project ideas is available on the [Jenkins website](https://jenkins.io/projects/gsoc/2020/project-ideas/).
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
-# mentoring
-Google Summer of Code, CommunityBridge, Outreachy and other mentoring projects 
+# CDF Mentoring Programs
+
+Google Summer of Code
+---------------------
+
+CDF is planning to participate in GSoC 2020.
```
Comment (oleg-nenashev) [inline README.md]:
```suggestion
CDF is planning to participate in Google Summer of Code 2020. 
This year CDF organizes GSoC for Spinnaker, Tekton, and Screwdriver.cd.
Jenkins project is a [separate GSoC organization](https://summerofcode.withgoogle.com/organizations/4945163270488064/).
Jenkins X also participates in GSoC under the Jenkins umbrella.
```

**Review by oleg-nenashev — COMMENTED** [review]
Some suggestions for Jenkins and Jenkins X

**Pull request merged**